src/software/gui/drawing/ball.h

```diff
@@ -21,31 +19,42 @@
  * @param ball The ball to draw
  * @param color The color to draw the ball's velocity
  */
-void drawBallVelocity(QGraphicsScene *scene, const Ball &ball, const QColor &color);
+void drawBallVelocity(QGraphicsScene *scene, const Point &position,
+                      const Vector &velocity, const QColor &color);
 
 /**
  * Draws the ball's position on the given scene.
  *
  * @param scene The scene to draw on
- * @param ball The ball to draw
+ * @param position The position of the ball
+ * @param height the height of the ball
```

Comment:
similairly, can we rename to `distance_from_ground`, same goes for other similar usages of "height" in this PR

Reply:
done, pretty sure I got everything